README.md

```diff
@@ -182,6 +182,19 @@ Static fields are added to the [Resource](https://opentelemetry.io/docs/concepts
 
 Fields that can change during the lifetime of the page are instead added to each span in a [SpanProcessor](https://opentelemetry.io/docs/specs/otel/trace/sdk/#span-processor).
 
+#### GlobalErrorsInstrumentationConfig
```

Comment:
Wasn't 100% sure where the attributes would fit in the doc, but I think this makes the most sense. I'm open to moving it though!